Sprint-1/exercises/random.js

```diff
@@ -2,6 +2,7 @@ const minimum = 1;
 const maximum = 100;
 
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
+console.log(num); // num represents a random number generated between 1 and 100 (minimum & maximum) : a random number is generated using Math.random() then brought into a whole integer using Math.floor()
```

Comment:
To test your understanding, how would you write an expression (without using any variable in the expression) that yields a random integer between -5 and 3 (including -5 and 3)?